en/docs/get-started/about-this-release.md

```diff
@@ -42,6 +42,7 @@ WSO2 IS 5.11.0 contains the following new features and enhancements:
 
 - /identity/connect/register API
 - OAuth 1.0
+- H2 Console
```

Comment:
Is this a deprecated feature or a removed feature?

Reply:
This feature will be removed from the product. Hence, should we add this under a different heading?

Reply:
Shall we give a small description reasoning out why this has been removed?
- Reported security issue
- Only used in the development time
- No real use case in the production environment

And ask to use standard database client software as the replacement.